hive/pom.xml

```diff
@@ -23,13 +23,18 @@
     <parent>
         <groupId>io.confluent</groupId>
         <artifactId>kafka-connect-storage-common-parent</artifactId>
-        <version>3.3.1-SNAPSHOT</version>
+        <version>3.3.2-SNAPSHOT</version>
     </parent>
 
     <artifactId>kafka-connect-storage-hive</artifactId>
     <packaging>jar</packaging>
     <name>kafka-connect-storage-hive</name>
 
+    <properties>
+        <commons.lang3.version>3.1</commons.lang3.version>
+        <kryo.version>2.22</kryo.version>
```

Comment:
Hi @wicknicks 
Did you forget to add the property for `avro.version`?

Reply:
It's defined here (https://github.com/confluentinc/common/blob/master/pom.xml#L48), so we can share it across all avro dependent repos.

Reply:
Yes, you are right. I missed it because I was depending on common v4.0.0 to build this.